src/main/java/core/basesyntax/FigureGenerator.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureGenerator {
+    public static Figures getRandomFigure() {
+        switch (new Random().nextInt(5)) {
```

Comment:
declare 5 as a constant, now it's magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names